vrecord

```diff
@@ -37,6 +37,25 @@ MPVOPTS+=(--load-scripts=no)
 MPVOPTS+=(--script "${SCRIPTDIR}/qcview.lua")
 MPVOPTS+=(--really-quiet)
 
+# Set variables for system differences
+SYSTEM_TEST="$(uname -s)"
+if [[ "${SYSTEM_TEST}" = "Darwin" ]] ; then
+    OS_TYPE="macOS"
+    DEFAULTFONT='/Library/Fonts/Andale Mono.ttf'
+    HARDWARE_REPORT=$(system_profiler SPHardwareDataType)
+    CORE_COUNT="$(_parse_report "Total Number of Cores" "${HARDWARE_REPORT}")"
```

Comment:
untested, but I suspect this would not work since you're using a function before it is defined

Reply:
yes! that would appear to be the case - thanks!